onnxruntime/contrib_ops/cuda/bert/attention_impl.cu

```diff
@@ -232,7 +250,7 @@ Status LaunchAttentionKernel(
     int batch_size,
     int sequence_length,
     int num_heads,
-    int head_size,
+    const int* qkv_head_size,
```

Comment:
It might be better to pass two parameters: qk_head_size and v_head_size for performance.